### DISCUSSION TITLE
DIY Flow Cell

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#723] [REM] [answered]
I was planning on reusing the flow cell from my Intellichem setup, but I spent a couple of hours yesterday just trying to get the rusty screws free so I could move it.  In about an hour, I'm breaking out the SawsAll and I'm likely to destroy it.  So, I'm thinking about a new one.

You said:d

"While you could certainly build your own from PVC there are advantages to using one of the pre-built ones out there."

Care to elaborate?  What are the advantages of the pre-built ones?

A clear disadvantage is cost.  Around here, they tend to freeze and I resent paying over $300 for a piece of injected plastic:

https://www.amazon.com/Pentair-Water-Pool-754000340-Swimming/dp/B00E4QC33S/ref=sr_1_28_sspa?hvadid=616991154887&hvdev=c&hvlocphy=9009678&hvnetw=g&hvqmt=b&hvrand=13263189701837390824&hvtargid=kwd-298861033419&hydadcr=26614_11715055&keywords=flow+cell&qid=1681990473&sr=8-28-spons&psc=1&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUExTDJNVEpZNENJTjFCJmVuY3J5cHRlZElkPUEwMDU3MTIzMklEQUU5NUU5VkNBNyZlbmNyeXB0ZWRBZElkPUEwMjE3NTY1QzBCRUUyMDgwSzYwJndpZGdldE5hbWU9c3BfbXRmJmFjdGlvbj1jbGlja1JlZGlyZWN0JmRvTm90TG9nQ2xpY2s9dHJ1ZQ==

Couldn't pvc pipe with threaded 3/4" T fittings work for the probe holders?  I see separate, inline flow switches for around $25.

Am I missing something, or should I run down to Lowe's and pick up some pvc and fittings?

**Comment by tagyoureit** [comment, marked as answer]
@rstrouse had the great idea of mounting the flow cell on the back of his Intellicenter and then covering the whole thing with a garbage bag.  The power supply creates just enough heat that it should keep the cell from melting.  But also, your pool should be running when temps are freezing and the path/flow of water should include the flow cell.

The advantages to using the pre-built ones are:
1. Visibility - you get to see what's happening
2. Less flow - You don't need 1/2" pvc to get the readings; diverting only 3/8" with flexible tubing is sufficient
3. Built in flow switch
4. Water path creates a vortex effect and reduces the amount of bubbles present which can affect readings
5. (maybe a dis-advantage) you can drill/expand the 1/2" holes into 3/4" holes for the professional grade probes

I'm not saying these are the strongest reasons in the world, and maybe not worth it for the cost, but I would definitely take a look at the placement of the probes (for heat) and include the probes in the freeze mode set up.

**Comment by davthomaspilot** [comment]
My Intellichem flow cell was in the path and the pumps do run when the temps are near (or below freezing).  Still, the water in the flow cell jar froze and broke the internal pipe that is used for the flow detect. 

That is a good idea--use the heat from the rpi and power supply inefficiency to warm the flow cell.  Heated pipe tape might work well too.

I was finally able to get my Intellichem off the concrete wall.  Basically, the screws got so hot from the SawsAll the plastic holes melted.  

It's going to be easier to salvage it than DIY another.   

Thanks!
